packages/components/src/widgets/TableChartWidget.tsx

```diff
@@ -86,9 +87,18 @@ widgetRegistry.register("TableChart", {
                   {row.map((item, k) => (
                     <td
                       key={k}
+                      tabIndex={1}
```

Comment:
This is very minimal, but does an (incredibly basic) first pass. We can improve a lot later.